server/src/main/java/org/spine3/server/SubscriptionService.java

```diff
@@ -0,0 +1,212 @@
+ *
+ * @author Alex Tymchenko
+ */
+public class SubscriptionService extends SubscriptionServiceGrpc.SubscriptionServiceImplBase {
+    private final ImmutableMap<TypeUrl, BoundedContext> typeToContextMap;
+
+    private SubscriptionService(Builder builder) {
+        this.typeToContextMap = builder.getBoundedContextMap();
+    }
+
+    public static Builder newBuilder() {
+        return new Builder();
+    }
+
+    @Override
+    @SuppressWarnings("RefusedBequest")     // as we override default implementation with `unimplemented` status.
+    public void subscribe(Topic topic, StreamObserver<Subscription> responseObserver) {
+        log().debug("Creating the subscription to a topic: {}", topic);
+
+        try {
+            final Target target = topic.getTarget();
+            final BoundedContext boundedContext = selectBoundedContext(target);
+            final Stand stand = boundedContext.getStand();
+
+            final Subscription subscription = stand.subscribe(target);
+
+            responseObserver.onNext(subscription);
+            responseObserver.onCompleted();
+        } catch (@SuppressWarnings("OverlyBroadCatchBlock") Exception e) {
+            log().error("Error processing subscription request", e);
+            responseObserver.onError(e);
+            responseObserver.onCompleted();
+        }
+    }
+
+    @Override
+    @SuppressWarnings("RefusedBequest")     // as we override default implementation with `unimplemented` status.
+    public void activate(final Subscription subscription, final StreamObserver<SubscriptionUpdate> responseObserver) {
+        log().debug("Activating the subscription: {}", subscription);
+
+        try {
+            final BoundedContext boundedContext = selectBoundedContext(subscription);
+
+            final Stand.StandUpdateCallback updateCallback = new Stand.StandUpdateCallback() {
+                @Override
+                public void onEntityStateUpdate(Any newEntityState) {
+                    Preconditions.checkNotNull(subscription);
+                    final SubscriptionUpdate update = SubscriptionUpdate.newBuilder()
+                                                                        .setSubscription(subscription)
+                                                                        .setResponse(Responses.ok())
+                                                                        .addUpdates(newEntityState)
+                                                                        .build();
+                    responseObserver.onNext(update);
+                }
+            };
+            final Stand targetStand = boundedContext.getStand();
+            targetStand.activate(subscription, updateCallback);
+
+        } catch (@SuppressWarnings("OverlyBroadCatchBlock") Exception e) {
+            log().error("Error activating the subscription", e);
+            responseObserver.onError(e);
+            responseObserver.onCompleted();
```

Comment:
We cannot have `onCompleted()` after `onError()` according to specs of `StreamObserver`.